app/models/report.rb

```diff
@@ -88,29 +99,24 @@ def unresolved_siblings?
   end
 
   def history
-    time_range = created_at..updated_at
-
-    sql = [
+    subquery = [
       Admin::ActionLog.where(
         target_type: 'Report',
-        target_id: id,
-        created_at: time_range
-      ).unscope(:order),
+        target_id: id
+      ).unscope(:order).arel,
 
       Admin::ActionLog.where(
         target_type: 'Account',
-        target_id: target_account_id,
-        created_at: time_range
-      ).unscope(:order),
+        target_id: target_account_id
+      ).unscope(:order).arel,
 
       Admin::ActionLog.where(
         target_type: 'Status',
-        target_id: status_ids,
-        created_at: time_range
-      ).unscope(:order),
-    ].map { |query| "(#{query.to_sql})" }.join(' UNION ALL ')
+        target_id: status_ids
```

Comment:
These include actions that date from before the report, I don't know whether that's deliberate or not. It can be useful, but also noisy or confusing.

Reply:
Deliberate, I think it's more useful to see an account has been acted on before...

Reply:
I think it can be confusing but it's probably not a big deal. We'll see I guess.